src/Movie/Movie.js

```diff
@@ -17,20 +18,21 @@ export default function Movie({title, releaseDate, averageRating, posterImage, d
             {/* <button className="on-image">Heart</button> */}
             <button className="on-image" onClick={toggleLikeButton}>
                 {!isLiked ? (
-                <svg className="heart on-image" fill="none" stroke="currentColor" stroke-width="1.5" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
-                <path stroke-linecap="round" stroke-linejoin="round" d="M21 8.25c0-2.485-2.099-4.5-4.688-4.5-1.935 0-3.597 1.126-4.312 2.733-.715-1.607-2.377-2.733-4.313-2.733C5.1 3.75 3 5.765 3 8.25c0 7.22 9 12 9 12s9-4.78 9-12z"></path>
+                <svg className="heart on-image" fill="none" stroke="currentColor" strokeWidth="1.5" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
```

Comment:
You may be able to import these SVGs from a different file to improve readability here